public/ACMW/css/index.css

```diff
@@ -125,10 +125,25 @@
 /*.dropdown-menu li a:hover {
 	background-color: red !important;
 } */
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+footer .container{
+	padding: 80px 120px 10px;
+}
+>>>>>>> replaced padding properties to single padding property
+=======
+footer .container{
+	padding: 80px 120px 10px;
+}
+>>>>>>> e26a74ba8763494f3845f4433410f1a7af2ff448
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)